### ISSUE TITLE
PyCharm 2021.1.1 fails to give parameter hinting

### ISSUE THREAD

**Issue by wuyuanyi135** [strawberry-graphql/strawberry-pycharm-plugin#5]
Ctrl+P shows no parameter hinting below
Ctrl+Q does not show anything.
![image](https://user-images.githubusercontent.com/11760870/117236653-183b5100-adf7-11eb-8949-9d049251add1.png)

**Comment by koxudaxi** [comment]
@wuyuanyi135 
I can reproduce the problem.

There is a related issue.
https://github.com/koxudaxi/pydantic-pycharm-plugin/issues/267#issuecomment-821832674
> PyCharm 2021.1 detects types of decorators. PyCharm 2020.3 doesn't work the feature.

I will fix it.

**Comment by koxudaxi** [comment]
But, `strawberry.type` doesn't have type annotation of return value :thinking: 

```python
def type(
    cls: Type = None,
    *,
    name: str = None,
    is_input: bool = False,
    is_interface: bool = False,
    description: str = None,
    federation: Optional[FederationTypeParams] = None,
):
```

**Comment by koxudaxi** [comment]
This line affects analyzing the type.
`cls: Type = None,`

**Comment by BryceBeagle** [comment]
@wuyuanyi135 I think this should be resolved now

**Comment by wuyuanyi135** [comment]
@BryceBeagle Thanks. Is there plan for a new release of the plugin?

**Comment by BryceBeagle** [comment]
@koxudaxi Does CD trigger when a release is tagged?

**Comment by koxudaxi** [comment]
@BryceBeagle 
CD is triggered when a tag is pushed.
And, CD creates a release.

**Comment by BryceBeagle** [comment]
Let's make a release for @wuyuanyi135 👍

**Comment by koxudaxi** [comment]
@wuyuanyi135 
https://github.com/strawberry-graphql/strawberry-pycharm-plugin/releases/tag/v0.0.8
You can download the compiled plugin on the page.
Also, The version will be published in the Jetbrains market after Jetbrains approves the plugin within two business days

**Comment by wuyuanyi135** [comment]
@koxudaxi @BryceBeagle  Many thanks 👍 ! I can confirm the issue is fixed in v0.0.8. Closing.

**Comment by patrick91** [comment]
This has been released/approved now on the store too 😊
